pages/privacy-notice-terms-conditions.js

```diff
@@ -29,6 +29,31 @@ import { getToken } from 'next-auth/jwt'
 
 export default function PrivacyCondition(props) {
   const t = props.locale === 'en' ? en : fr
+  
```

Comment:
This shouldn't be being added. This was just removed from all pages in another PR that was merged to dev recently. You may need to pull from dev locally and then do a git merge on this branch again

Reply:
good catch. removed that section.